File: ABC/ABC139/B/main.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {

  int A, B;
  cin >> A >> B;

  int count = 0;
  int sum = A;

  while (sum < B) {
    count++;
    sum = sum - 1 + A;
  }

  if (B == 1) {
    count = 0;
  }

  cout << count << endl;

}

File: ABC/ABC139/B/main.java
import java.util.*;

public class Main {

  public static void main (String[] args) {

    Scanner sc = new Scanner(System.in);

    int A = sc.nextInt();
    int B = sc.nextInt();

    sc.close();

    int count = 1;
    int sum = A;

    while (sum < B) {
      count++;
      sum = sum - 1 + A;
    }

    if (B == 1){
      count = 0;
    }
    
    System.out.println(count);

  }

}